Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'hiredis'
 gem 'httparty'
 gem 'lograge'
 gem 'newrelic_rpm'
+gem 'nokogiri' # required for premailer-rails
```

Comment:
Is it necessary to specify `nokogiri` in the Gemfile, even though it's already installed? We have several gems that depend on it.